ecologits/impacts/llm.py

```diff
@@ -28,7 +28,55 @@
 
 HARDWARE_LIFESPAN = 5 * 365 * 24 * 60 * 60
 
-DATACENTER_PUE = 1.2
+AI_PROVIDER_PUE = {
+    "anthropic"	: 1.09,
+    "mistralai"	: 1.26,
+    "cohere"	: 1.15,
+    "databricks" : 1.18,
+    "meta"	: 1.09,
+    "azureopenai" : 1.18, #treated the same way as OpenAI
+    "huggingface_hub" : 1.15,
+    "google" : 1.09,
+    "microsoft"	: 1.18,
+    "openai" : 1.18
+    #"litellm" : 1.15 #need a way to identify provider from model inputed
+}
+
+AI_PROVIDER_WUE_ON_SITE = {
+    "anthropic"	: 0.916,
+    "mistralai"	: 0.37, #2024
+    "cohere"	: 0.18, #2023
+    "databricks" : 0.49, #2022
+    "meta"	: 0.18,    # L/kWh, 2023
+    "azureopenai" : 0.49, #2022 #treated the same way as OpenAI
+    "huggingface_hub" : 0.18, #2023
+    "google" : 0.916,
+    "microsoft"	: 0.49, #2022
+    "openai" : 0.49, #2022
+    "litellm" : 0.18, #2023 #need a way to identify provider from model inputed
+}
```

Comment:
This should preferably be in the `tracers/utils.py` module. Match the provider with these mappings to then input the correct `datacenter_pue` and `datacenter_wue` values in the `compute_llm_impacts` function.